editor/src/components/canvas/canvas-strategies/canvas-strategies.spec.browser2.tsx

```diff
@@ -290,10 +290,10 @@ describe('Strategy Fitness', () => {
 
     const interactionSession: InteractionSession = {
       ...createMouseInteractionForTests(
-        canvasPoint({ x: 0, y: 0 }),
+        canvasPoint({ x: 10, y: 10 }),
```

Comment:
what is the reason for this change?

Reply:
This test was previously faking the `startingTargetParentsToFilterOut` as part of creating the `InteractionSession`, but that is no longer applicable as we calculate those via the reparent metastrategy. Because this test also used `0,0` as the drag's starting point, that was causing the `getReparentTargetUnified` to pick up the the canvas as the target to filter out, meaning no reparent strategy was applicable. Moving the starting point inside the actual dragged element ensure the correct target is ignored.